Building out of the box

Created new xcode project, followed instructions per readme and get the follow error.

```
OctoKit-Prefix.pch:18:10: fatal error: 'Mantle/Mantle.h' file not found
        #import <Mantle/Mantle.h>
                ^
1 error generated.
```

I attempted to fix this issue myself by adding `External/Mantle` to the Octokit.xcodeproj build settings `Header Search Paths`. Then the process is started again with `ReactiveCocoa`.

```
OctoKit-Prefix.pch:19:10: fatal error: 'ReactiveCocoa/ReactiveCocoa.h' file not found
        #import <ReactiveCocoa/ReactiveCocoa.h>
                ^
1 error generated.
```

Add `External/ReactiveCocoa/ReactiveCocoaFramework` to Octokit.xcodeproj build settings `Header Search Paths`. Then finally get this error I cant resolve:

```
OctoKit-Prefix.pch:19:
In file included from External/ReactiveCocoa/ReactiveCocoaFramework/ReactiveCocoa/ReactiveCocoa.h:12:
External/ReactiveCocoa/ReactiveCocoaFramework/ReactiveCocoa/NSObject+RACPropertySubscribing.h:10:9: fatal error: 'ReactiveCocoa/EXTKeyPathCoding.h' file not found
#import <ReactiveCocoa/EXTKeyPathCoding.h>
        ^
1 error generated.
```

I haven't worked with xcode much, what am I missing?

--- a/README.md
+++ b/README.md
@@ -17,8 +17,8 @@ To add OctoKit to your application:
  1. Add the OctoKit repository as a submodule of your application's
     repository.
  1. Run `script/bootstrap` from within the OctoKit folder.
- 1. Drag and drop `OctoKit.xcodeproj` into your application's Xcode project or
-    workspace.
+ 1. Drag and drop `OctoKit.xcodeproj` into the top-level of your application's workspace (or into the top-level of your project file to create a new workspace).
+ 1. Drag and drop `ReactiveCocoa.xcodeproj` into the top level of your workspace. (This project can be found in the `External` folder).
  1. On the "Build Phases" tab of your application target, add OctoKit to the "Link
     Binary With Libraries" phase.
     * **On iOS**, add `libOctoKit.a`.

